sdk_container/src/third_party/coreos-overlay/sys-apps/ignition/files/0003-mod-add-flatcar-ignition-0.36.2.patch

```diff
@@ -1,47 +1,53 @@
-From 99d75703026de8787b4bb83e77bd07d1f6d730b9 Mon Sep 17 00:00:00 2001
+From af61357bfc985942f51c79787e282ff25399bf49 Mon Sep 17 00:00:00 2001
 From: Mathieu Tortuyaux <mtortuyaux@microsoft.com>
-Date: Wed, 26 Mar 2025 15:10:08 +0100
+Date: Tue, 12 Aug 2025 15:10:05 +0200
 Subject: [PATCH 03/20] mod: add flatcar/ignition@0.36.2
 
 it's required for Ignition conversion.
 
 Signed-off-by: Mathieu Tortuyaux <mtortuyaux@microsoft.com>
 ---
- go.mod | 4 ++++
- 1 file changed, 4 insertions(+)
+ go.mod | 6 +++++-
+ 1 file changed, 5 insertions(+), 1 deletion(-)
 
 diff --git a/go.mod b/go.mod
-index f94bbafa..2afaf4ae 100644
+index c4aca58f..be20c337 100644
 --- a/go.mod
 +++ b/go.mod
-@@ -14,6 +14,7 @@ require (
+@@ -1,4 +1,4 @@
+-module github.com/coreos/ignition/v2
++module github.com/flatcar/ignition/v2
```

Comment:
This change probably should remain being a part of the `sed s/coreos/flatcar/` patch, right?